src/binding/fortran/mpif_h/mpi_fortimpl.h

```diff
@@ -7,6 +7,10 @@
 #define MPI_FORTIMPL_H_INCLUDED
 
 #include "mpichconf.h"
+#include "mpi.h"
```

Comment:
I think this is probably good. Looks like we need to `#include <sys/types.h>` somewhere to get `ssize_t` on BSD and macOS.

Reply:
Added.